aip/general/0126.md

```diff
@@ -55,12 +55,18 @@ message Book {
   - An exception to this rule is if there is a clearly useful zero value. In
     particular, if an enum needs to present an `UNKNOWN`, it is usually clearer
     and more useful for it to be a zero value rather than having both.
+- The other values **should not** be prefixed by the name of the enum itself.
+  This generally requires users to write `MyState.MYSTATE_ACTIVE` in their
+  code, which is unnecessarily verbose.
+  - Note that some languages (including C++) hoist enum values into the parent
+    namespace, which can result in conflicts for enums with the same values. To
```

Comment:
Nit: tl;dr what's the equivalent of a C++ namespace in protobuf?

Would it be worth relating a C++ namespace to the equivalent in protobuf? I've not written a line of C++ in my life, and IDK what to look out for in my protos that would precipitate this collision. Is it just proto package? Is it proto element level (i.e. enums defined at the file level, or within a `message`, etc.)? Is it at all related to the language options used for code generation?

Reply:
I've never written a line of C++ either:) But it looks like the answer to this question is "proto package": https://developers.google.com/protocol-buffers/docs/reference/cpp-generated#package

Updated the PR.